### PR TITLE
xlwt version of colouring in

### PR DESCRIPTION
**Pull request by scraperdragon**

### CODE REVIEW

File: databaker/bake.py [hunk 1]
```diff
@@ -70,8 +74,11 @@ class Opt(object):
     options = docopt(__doc__, version='databaker {}'.format(__version__))
     xls_files = options['<spreadsheets>']
     recipe_file = options['<recipe>']
-    timing = options['--timing']
-    csv_file = 'out.csv'
+    timing = not options['--notiming']
+    preview = options['--preview']
+    preview_filename = "preview-{fn}-{recipe}.xls"
```
Comment (frabcus) [inline databaker/bake.py]: Maybe call "fn" something like "excel" or "spreadsheet"?

File: databaker/bake.py [hunk 2]
```diff
@@ -197,28 +204,64 @@ def update(self, count):
...
+                    for ob in obs:
+                        writer.get_sheet(tab.index).write(ob.y, ob.x, ob.value,
+                            xlwt.easyxf('pattern: pattern solid, fore-colour {}'.format(colourlist[OBS])))
+
+
+        if Opt.csv:
+            obs_count = len(obs)
+            progress = Progress(obs_count, 'Tab {}'.format(tab_num + 1))
+            for ob_num, ob in enumerate(obs):  # TODO use const
+                csv.handle_observation(ob)
```
Comment (frabcus) [inline databaker/bake.py]: At some point maybe explicitly separate ONS code and give it names that it is ONS specific?

File: databaker/bake.py [hunk 3]
```diff
@@ -197,28 +204,64 @@ def update(self, count):
...
+    if Opt.csv:
+        csv.footer()
+
+
+colourlist = {OBS: "blue",
+              DATAMARKER: "blue_gray",
+              TIME: "bright_green",
+              TIMEUNIT: "cyan_ega",
+              GEOG: "coral",
+              0: "brown",
```
Comment (frabcus) [inline databaker/bake.py]: I get purple for my first dimension. Is 0 not being used?

File: databaker/bake.py (hunk 2 above)
Comment (scraperdragon) [inline databaker/bake.py]: The ONS specific stuff is in `TechnicalCSV` class

File: databaker/bake.py (hunk 3 above)
Comment (scraperdragon) [inline databaker/bake.py]: Fixed

File: databaker/bake.py (hunk 1 above)
Comment (scraperdragon) [inline databaker/bake.py]: Fixed.

**Pull request merged**